staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1/api.proto

```diff
@@ -4,16 +4,6 @@ syntax = "proto3";
 package pluginregistration; // This should have been v1.
 option go_package = "k8s.io/kubelet/pkg/apis/pluginregistration/v1";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
-
-option (gogoproto.goproto_stringer_all) = false;
-option (gogoproto.stringer_all) =  true;
-option (gogoproto.goproto_getters_all) = true;
-option (gogoproto.marshaler_all) = true;
-option (gogoproto.sizer_all) = true;
-option (gogoproto.unmarshaler_all) = true;
-option (gogoproto.goproto_unrecognized_all) = false;
```

Comment:
need to look up what each of these gogo-specific options does, and make sure sig-node is ok with whatever they did no longer being done with standard proto

it might also be helpful to look up when and why these were added to this file ... it's entirely possible they were just copied from some other starting point without being specifically needed / desired

Reply:
For the various graduations it looks like they were just copied over, while being introduced in c440246966db13. I also _assume_ that the changes are fine because we have no required code changes for any conversion. Clients may have, though.

cc @kubernetes/sig-node-api-reviews

Reply:
cc @saad-ali @ffromani @klueska @pohly :point_up:

Reply:
the "unrecognized" field generation was the only thing I was really concerned about, but since the kubelet doesn't *persist* this data anywhere, I don't think that has any impact (something like kube-apiserver persisting unrecognized fields into etcd would be really problematic, but that's not an issue here)